maintainers/maintainer-list.nix

```diff
@@ -13249,6 +13249,12 @@
     githubId = 102685;
     name = "Thomas Friese";
   };
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
I move this into its own commit. As for sorting it alphabetically, I believe it was already sorted correctly. Is it sorted by something other than the attribute name?